pkg/apis/ibmcloudprovider/v1/ibmcloudproviderconfig_types.go

```diff
@@ -36,6 +36,9 @@ type IBMCloudMachineProviderSpec struct {
 	// VPC name where the instance will be created
 	VPC string `json:"vpc"`
 
+	// BootVolume is the machine's boot volume details
```

Comment:
what happens when this is omitted?

like, what is the default behavior, should we document that here?

Reply:
Sure, I can add a comment about that.

Reply:
Updated, hopefully that's more in line to what you were thinking.